src/end.sol

```diff
@@ -148,6 +148,11 @@ interface SpotLike {
     all auctions are in the reverse (`dent`) phase. There are two ways
     of ensuring this:
```

Comment:
This line references two options on old line 151 and 158.  The new comment seems to be wedged in the middle

Reply:
Yeah, I'd suggest moving the new paragraph right after the `- backing collateral taken`